src/devices/Arduino/samples/ApiChecker/RgbLedTest.cs

```diff
@@ -57,8 +55,7 @@ public void DoTest()
             float blueCorrection = 0.95f;
             while (!Console.KeyAvailable)
             {
-                var hsv = new Hsv(angle, 1.0f, 1.0f);
-                var rgb = converter.ToRgb(hsv);
+                Color rgb = Color.Blue;
```

Comment:
isn't this sample just displaying blue rather than color changing over time?

Reply:
Yes, this will be fixed with the next PR.